test/browser_tests/spec_suites/about-deputy-director.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var TheAboutDirectorPage = require( '../page_objects/page_about-director.js' );
+
+describe( 'The About Deputy Director Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new TheAboutDirectorPage( );
+    page.get( 'deputyDirector' );
+  } );
+
+  it( 'should properly load in a browser',
+    function() {
+      expect( page.pageTitle() ).toBe( 'About Meredith Fuchs' );
+    }
+  );
+
+  it( 'should have a side nav',
+    function() {
+      expect( page.sideNav.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the Deputy Director\'s bio',
+    function() {
+      expect( page.directorBioSummary.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the Deputy Director\'s summary',
+    function() {
+      expect( page.directorBioSummary.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the Deputy Director\'s title',
+    function() {
+      expect( page.directorTitle.getText() )
+      .toEqual( 'Deputy Director Meredith Fuchs' );
+    }
+  );
+
+  it( 'should include the Deputy Director\'s image',
+    function() {
+      expect( page.directorImage.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the More Info section',
+    function() {
+      expect( page.moreInfo.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the More Info title',
+    function() {
+      expect( page.moreInfoTitle.getText() )
+      .toEqual( 'More information about Meredith Fuchs' );
+    }
+  );
+
+  it( 'should include More Info items',
+    function() {
+      expect( page.moreInfoItems.count() ).toEqual( 3 );
+    }
+  );
+
+  it( 'should include the bio download link',
+    function() {
+      expect( page.bioDownload.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the Deputy Director\'s high resolution image',
+    function() {
+      var highResImageDownload = page.mediaDownloads.get( 0 );
```

Comment:
This is correct because the `Deputy Director’s` page doesn't have a `Biography` link.